server/src/main/java/org/opensearch/index/compositeindex/CompositeIndexConstants.java

```diff
@@ -19,13 +19,13 @@ public class CompositeIndexConstants {
     public static final long COMPOSITE_FIELD_MARKER = 0xC0950513F1E1DL; // Composite Field
 
     /**
-     * Represents the key to fetch number of non-star aggregated segment documents.
+     * Represents the key to fetch number of non-star aggregated records/entries
```

Comment:
nit: it's fine to remove records if we are not using this term in the code for now